package.json

```diff
@@ -51,7 +51,7 @@
     "cross-fetch": "3.1.4",
     "dotenv": "10.0.0",
     "emittery": "0.8.1",
-    "type-fest": "1.2.1"
+    "type-fest": "^2.12.0"
```

Comment:
happy to upgrade if we need it but can we keep it pinned?

Reply:
btw what does this new version provide that we needed?

Reply:
Sorry guys, my bad, it doesn't provide anything that we need.

Just thought we could use the latest, but happy to put it back, done with https://github.com/yearn/yearn-sdk/pull/243/commits/6868d1868d41bb1255bb79bb0cfa990966e93730